apps/02RockPaperScissors.js

```diff
@@ -9,15 +9,44 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
-  // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!"
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+
+  } else if (hand1 === "paper") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+
+  } else if (hand1 === "scissors") {
+
+    if (hand2 === "rock") {
+      return "Hand two wins!";
+    } else {
+      return "Hand one wins!";
+    }
+
+  } else {
+    return "Unavailable option selected."
+  }
```

Comment:
Nice! Input scrubbing is a great practice. You never know what kind of text your end-user might try to send through your application :)